### ISSUE TITLE
Compilation fails on FreeBSD

### ISSUE THREAD

**Issue by interkosmos** [libgit2/libgit2#5627]
### Reproduction steps
On FreeBSD, run:
```
$ git clone https://github.com/libgit2/libgit2
$ cd libgit2/
$ mkdir build && cd build/
$ cmake ..
$ cmake --build .
[ 27%] Built target git2internal
[ 28%] Built target ntlmclient
[ 28%] Built target http-parser
[ 28%] Built target git2
[ 28%] Building C object tests/CMakeFiles/libgit2_clar.dir/clar.c.o
In file included from /tmp/libgit2/tests/clar.c:786:
/tmp/libgit2/tests/clar/fs.h:277:11: fatal error: 'copyfile.h' file not found
# include <copyfile.h>
          ^~~~~~~~~~~~
1 error generated.
gmake[2]: *** [tests/CMakeFiles/libgit2_clar.dir/build.make:582: tests/CMakeFiles/libgit2_clar.dir/clar.c.o] Error 1
gmake[1]: *** [CMakeFiles/Makefile2:289: tests/CMakeFiles/libgit2_clar.dir/all] Error 2
gmake: *** [Makefile:160: all] Error 2
```       

### Expected behavior
Compiles on FreeBSD.

### Actual behavior
Compilation fails with error stating that `copyfile.h` is missing.

### Version of libgit2 (release number or SHA1)
1e987525b3bd86fc29e7ff67d973861efd22faf7

### Operating system(s) tested
FreeBSD 12.1 (amd64)

**Comment by pks-t** [comment]
This has been fixed via #5628, right? I'm thus closing the issue, thanks a lot for your report and fix!

**Comment by interkosmos** [comment]
Yes.
